app/controllers/admin/sales/invoices.js

```diff
@@ -39,7 +38,11 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
         name      : 'Status',
         valuePath : 'status'
       }
+      //{
```

Comment:
Codacy found an issue: [Expected space or tab after '//' in comment.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6173716)